src/types.ts

```diff
@@ -68,36 +67,39 @@ export interface ConsentHookActions {
 export type ConsentState = CookieTypes;
 
 export interface CookieConsentHookState extends ConsentHookActions {
-  cookies: CookieWrapper;
+  acceptCookies: (cookies: CookieTypes, options?: AcceptCookiesOptions) => void;
+  cookies?: CookieWrapper;
   consent: ConsentState;
 }
 
 export interface StorageConsentHookState extends ConsentHookActions {
+  acceptCookies: (cookies: CookieTypes) => void;
   storage: Storage;
   consent: ConsentState;
 }
 
-export type UseCookieConsentType = (
-  options?: CookieConsentOptions
-) => ConsentHookStateType;
+export type UseCookieConsentHookType = (
+  options?: CookieConsentOptionsCookie
+) => CookieConsentHookState;
+
+export type UseStorageConsentHookType = (
+  options?: CookieConsentOptionsStorage
+) => StorageConsentHookState;
```

Comment:
Maybe also call the options in the same manner as the name of these types and their return types?